### PR TITLE
mobilecoind API: Get -> Create, Read -> Parse

### PR DESCRIPTION
**Pull request by eranrund**
### Motivation

The verbs related to b58 code stuff were "Get" for creating a b58 code and "Read" for parsing one - which are suboptimal. This PR moves to Create and Parse which are unambiguous.

### In this PR
* Whole bunch of renames

### CODE REVIEW

File: mobilecoind-json/src/bin/main.rs [hunk 1]
```diff
@@ -641,10 +641,10 @@ fn main() {
                 balance,
                 utxos,
                 public_address,
-                get_request_code,
-                read_request_code,
-                get_address_code,
-                read_address_code,
```
Comment (sugargoat) [inline mobilecoind-json/src/bin/main.rs:647]: I think there will need to be README updates as well

Comment (eranrund) [reply mobilecoind-json/src/bin/main.rs:647]: These are function names. The URLs haven't changed.

Comment (sugargoat) [reply mobilecoind-json/src/bin/main.rs:647]: Ah I see - sorry, I think you just went through this same comment loop on another PR today as well!

**Review by sugargoat — APPROVED** [review]

**Review by rjwalters — APPROVED** [review]
Do you also want to bump the pypi version? See #461

**Comment by eranrund** [comment]
> Do you also want to bump the pypi version? See #461

I bumped the version to 0.2 (since this is a breaking change), and will merge it when CI lets me. Can you please rebase https://github.com/mobilecoinofficial/mobilecoin/pull/461/files after that and push the release to PyPI?

**Pull request merged**